packages/wdio-junit-reporter/src/index.ts

```diff
@@ -230,19 +245,11 @@ class JunitReporter extends WDIOReporter {
             if (suiteKey.match(/^"before all"/)) {
                 continue
             }
-
-            // there should only be one spec file per runner so we can safely take the first element of the array
-            const specFileName = runner.specs[0]
-            const suite = this.suites[suiteKey]
-
-            if (isCucumberFrameworkRunner) {
-                builder = this._addCucumberFeatureToBuilder(builder, runner, specFileName, suite)
-            } else {
-                builder = this._addSuiteToBuilder(builder, runner, specFileName, suite)
-            }
+            let suite = this.suites[suiteKey]
+            if (isCucumberFrameworkRunner && suite.type === type) builder = this._addCucumberFeatureToBuilder(builder, runner, specFileName, suite)
```

Comment:
Can you change this to

```js
if (isCucumberFrameworkRunner && suite.type === type) {
  builder = this._addCucumberFeatureToBuilder(builder, runner, specFileName, suite)
} else if (!isCucumberFrameworkRunner) {
  builder = this._addSuiteToBuilder(builder, runner, specFileName, suite)
}
```

Reply:
@wswebcreation I will change, but just checking, we can also write if/else if like what I did. Because, there is only one statement for if and else if